### PR TITLE
reg form added

### PR DESCRIPTION
**Pull request by rkchalasani**

### CODE REVIEW

File: reg/reg.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.regmain{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: left;
+  font-family: "metrolight", "sans-serif";
+}
+.reginput1{
```
Comment (Maharshiharsha72) [inline reg/reg.css:9]: improper naming , reginput1 is the not the right naming convention

File: reg/reg.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  border: 2px solid rgb(0, 0, 0);
+  border-radius: 5px;
+  padding-left: 10px;
+  background-color: transparent;
+}
+.regname{
+    font-family: "metrolight", "sans-serif";
+  color: black;
+}
+.drop{
```
Comment (Maharshiharsha72) [inline reg/reg.css:21]: naming convention is not good  "drop"

File: reg/reg.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    justify-self: center;
+    display: flex;
+    flex-direction: column;
+    /* text-align: left; */
+    gap: 1rem;
+    border: 2px solid black;
+    width: 700px;
+}
+
+.reginput2{
```
Comment (Maharshiharsha72) [inline reg/reg.css:36]: naming problem

File: reg/reg.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (Maharshiharsha72) [inline reg/reg.html:3]: you could use better name for your html file

File: reg/reg.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  var fmsg = document.form.msg.value;
+  var int = new Array();
+  var fint = document.getElementsByName("Interests");
+  for (var x of fint) {
+    if (x.checked) {
+      int.push(x.value);
+    }
+  }
+  const message = function () 
+  {
```
Comment (Maharshiharsha72) [inline reg/reg.js:20]: where can i find the json object of the user ???

File: reg/reg.html [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+              <option value="monthly">monthly</option>
+              <option value="yearly">yearly</option>
+              <option value="weekly">weekly</option>
+            </select>
+          </div>
+          <p>Message</p>
+          <textarea
+            class="regmsg"
+ 
+
```
Comment (Maharshiharsha72) [inline reg/reg.html:88]: why there are few blanks in the code  , i mean empty lines

File: reg/reg.html [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="reg.css">
+</head>
+<body>
+  <div>
+  <div>
+
```
Comment (Maharshiharsha72) [inline reg/reg.html:13]: why empty lines ?

File: reg/reg.html [hunk 8]
```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Maharshiharsha72) [inline reg/reg.html:7]: Title can be relevant to the program that we have

File: reg/reg.css [hunk 9]
```diff
@@ -0,0 +1,63 @@
+.regmain{
```
Comment (Maharshiharsha72) [inline reg/reg.css:1]: regmain can be named properly

File: reg/reg.css (hunk 3 above)
Comment (rkchalasani) [reply reg/reg.css:36]: yes maharshi i will work on my naming conventions

File: reg/reg.css (hunk 9 above)
Comment (rkchalasani) [reply reg/reg.css:1]: yes maharshi i will work on my naming conventions

**Pull request merged**